aip-console-jenkins/README.md

```diff
@@ -136,23 +137,30 @@ You can manually add environment variables to a build or use global jenkins envi
 
 ## Other Topics
 
-#### Authentication with username (AIP Console Standalone)
+[//]: # (#### Authentication with username &#40;AIP Console Standalone&#41;)
```

Comment:
`[//]: # ` is that a comment ? I think rather than comment this section, you should just remove it. For v2, users can refer to the v2 branch (which will be deprecated sooner or later)

Reply:
I think we might add support for it, we have the options in the cli but it doesn't work.